src/app/user/user-account/user-account.component.ts

```diff
@@ -16,7 +16,7 @@
  */
 
 import { Component, OnInit } from '@angular/core';
-
+import { ChangePasswordService } from '../../service/change-password.service';
```

Comment:
Ẁe are sorting our scripts by functional modules (i.e. `Navigation`, `User`, ...) rather than into `service`s and `component`s. Thereby, everything that is related and interacting with each other (e.g. everything having to do with the user) is in the same folder, easy to find, and independent of other code.